app/models/sqs.go

```diff
@@ -222,6 +223,74 @@ func (r *SendMessageRequest) SetAttributesFromForm(values url.Values) {
 	}
 }
 
+func NewSendMessageBatchRequest() *SendMessageBatchRequest {
+	return &SendMessageBatchRequest{}
+}
+
+type SendMessageBatchRequest struct {
+	Entries  []SendMessageBatchRequestEntry
+	QueueUrl string
+}
+
+func (r *SendMessageBatchRequest) SetAttributesFromForm(values url.Values) {
+	for key := range values {
+
+		keySegments := strings.Split(key, ".")
+		//If keysegments is 3 or less, there is no attribute value like: (Entries.1.MessageBody)
+		if len(keySegments) <= 3 {
+			continue
+		}
+
+		// From like a value(Entries.1.MessageAttributes.1.Name), get the entry index and the attribute index.
+		entryIndex, err1 := strconv.Atoi(keySegments[1])
```

Comment:
I'm sorry, can we have a comment that mentions the pattern you're parsing at this index, and then another one for the index below?  This will be difficult to consume and remember in the long term and the documentation is poor at describing it.  

It took me a while in searching to figure out what we were going for (no fault of yours!), and I'd like to avoid that 30 minute search down the road.

Reply:
Even just something like the below would be amazing.  Thank you!
```
Both patterns below are supported here.
// strconv.Atoi(keySegments[1] - targets the index value in pattern: `Entries.1.MessageBody`
// strconv.Atoi(keySegments[3] - targets the index value in pattern: `Entries.1.MessageAttributes.1.Name`
```